src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -218,19 +213,20 @@ export default {
 			return this.conversation.objectType === CONVERSATION.OBJECT_TYPE.BREAKOUT_ROOM
 		},
 
-		hasBotV1API() {
-			return getCapabilities()?.spreed?.features?.includes('bots-v1')
+		supportBotsV1() {
+			return hasTalkFeature(this.token, 'bots-v1')
 		},
 
 		canConfigureBreakoutRooms() {
-			const breakoutRoomsEnabled = getCapabilities()?.spreed?.config?.call?.['breakout-rooms'] || false
 			return this.canFullModerate
-				&& breakoutRoomsEnabled
+				&& (getTalkConfig(this.token, 'call', 'breakout-rooms') || false)
```

Comment:
`|| false` are not needed in Boolean evaluation, it will return a falsy value `undefined`

Reply:
Decided not to change that. Though we could improve it later, I don't see a need for it (if works, don't touch)